### PR TITLE
[System.Diagnostics] Return dummy value from Process.SessionId.

### PR DESCRIPTION
**Pull request by madewokherd**

### CODE REVIEW

**Comment by akoeplinger** [comment]
Why do need this?

**Comment by madewokherd** [comment]
I'm hoping it'll help with an application (reported at https://bugs.winehq.org/show_bug.cgi?id=40338) that uses this.

File: mcs/class/System/System.Diagnostics/Process.cs [hunk 1]
```diff
@@ -302,7 +302,7 @@ ProcessModule[] GetModules_internal (SafeProcessHandle handle)
 		[DesignerSerializationVisibility (DesignerSerializationVisibility.Hidden)]
 		[MonitoringDescription ("The session ID for this process.")]
 		public int SessionId {
-			get { throw new NotImplementedException (); }
+			get { return 4; }
```
Comment (akoeplinger) [inline mcs/class/System/System.Diagnostics/Process.cs]: Could you please change this to 0? Session 0 is the console session, so this should map most closely to Mono's use cases :)

Comment (madewokherd) [inline mcs/class/System/System.Diagnostics/Process.cs]: Are you sure it's possible for this API to return 0 on Windows? When I test it on a Windows machine without any remoting involved, I always get a positive integer.

Comment (akoeplinger) [inline mcs/class/System/System.Diagnostics/Process.cs]: Yes. You can check in Task Manager if you enable the Session ID column, a lot of the services run in Session 0.

**Comment by madewokherd** [comment]
Updated to return 0.

**Comment by akoeplinger** [comment]
LGTM, thanks.

**Pull request merged**